spanner/integration_test.go

```diff
@@ -4331,6 +4331,8 @@ func TestIntegration_GFE_Latency(t *testing.T) {
 
 // TestIntegration_DropDatabaseProtection tests the drop database protection feature
 func TestIntegration_DropDatabaseProtection(t *testing.T) {
+	skipEmulatorTest(t)
+	skipUnsupportedPGTest(t)
```

Comment:
I think PG databases are supported by this feature?

Reply:
Yes. Removed skipping PG test for drop database protection